docs/conf.py

```diff
@@ -116,7 +117,7 @@ def _git_version():
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
+# html_theme = 'default'
```

Comment:
Sphinx actually gave a warning that default is being renamed to classic, and that alabaster is the new recommended? (Right now it still hasn't converted but there are warnings).

As far as how RTD works, http://read-the-docs.readthedocs.org/en/latest/theme.html, the docs say that the RTD theme will apply when not specified.